sample/src/main/java/com/patrykandpatryk/vico/sample/ui/ChartPager.kt

```diff
@@ -93,8 +97,11 @@ internal fun ChartPager(
             ) {
                 SwipeHint(
                     currentPage = state.currentValue + 1,
-                    pageCount = itemCount,
-                )
+                    pageCount = itemCount
+                ) { direction: Int ->
+                    val newState = (state.currentValue + direction).coerceIn(0 until itemCount)
+                    scope.launch { state.animateTo(newState) }
```

Comment:
I think it would be preferable for `animateTo` not to be called at all when `state.currentValue + direction` is outside of the `0 until itemCount` range. This would reduce the number of operations.

```suggestion
                    scope.launch {
                        (state.currentValue + direction)
                            .takeIf { it in 0 until itemCount }
                            ?.let { state.animateTo(it) }
                    }
```